<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCategory</class>
 <widget class="QDialog" name="AddCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>576</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>576</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="ACCanselButton">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>221</y>
     <width>165</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Cansel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACCreateCategoryButton">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>221</y>
     <width>165</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Create Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>76</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 43, 66);</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ACCategoryTitleLEdit">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>107</y>
     <width>469</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
